src/pages/dashboard/unstake-banner/unstake-banner.tsx

```diff
@@ -26,6 +31,24 @@ const UnstakeBanner = () => {
     }
   };
 
+  const handleUnstakeAndWithdraw = async () => {
+    if (!api3Pool) return;
+    const [err, tx] = await go(api3Pool.unstakeAndWithdraw());
```

Comment:
😉 
```suggestion
    const tx = await handleTransactionError(api3Pool.unstakeAndWithdraw());
```

Reply:
Yeah, but I could used it because `handleTransactionError` was in a different branch :)